piker/clearing/_paper_engine.py

```diff
@@ -530,7 +535,7 @@ async def trades_dialogue(
 
     ):
 
-        with open_pps(broker, 'paper', False) as table:
+        with open_pps(broker, 'paper') as table:
```

Comment:
not quite following why you're not writing immediately here any more?

Reply:
I'm pretty sure that `open_pps` call is just meant to load the `pps.toml` file into the `_positions` module var and then iterate on it to send position messages to the ems.